source/manifest.json

```diff
@@ -1,4 +1,10 @@
 {
+  "applications": {
+    "gecko": {
+      "id": "theoldreader@knyar",
+      "strict_min_version": "53.0"
+    }
+  },
```

Comment:
Do you already have a build script for Chrome? Can you add it here? To be honest, I have no experience with Chrome add-ons packaging.